### ISSUE TITLE
Focusable elements in the viewport cannot be focused via sequential keyboard navigation

### ISSUE THREAD

**Issue by danielwiehl** [SchweizerischeBundesbahnen/scion-toolkit#108]
## Describe the bug
If an element located in front of the viewport has focus and the tab key is pressed, focusable elements of slotted viewport content are skipped. This bug was introduced by issue #97 where we changed the viewport encapsulation to `ShadowDom`.

## To Reproduce

1. Add an input element before the viewport
2. Add an input element to the viewport
4. Add an input element after the viewport
5. Focus the input element before the viewport
5. Press the Tap key
6. See that the input element after the viewport has focus

## Expected behavior
The input element inside the viewport has focus.

## Example
```html
<input>
<sci-viewport>
  <input>
</sci-viewport>
<input>
````
